### ISSUE TITLE
[BUG] SLCs don't work (either not reaching chain or are not being attested)

### ISSUE THREAD

**Issue by verabehr** [VolumeFi/paloma#1875]
1. some chains for which we have no data yet (ETH, BNB, OP, Polygon, Blast, Base)
2. some chains for which SLCs reach the chain, but are failing to be attested (Arb)
```
{"level":"warn","module":"server","tx_hash":"0x85670a939cb9ee32430b41f533bf8ad5492b6c73fd46cee886e6ca8cb6bc4332","valset_id":1352,"time":"2024-07-15T17:19:15Z","message":"SubmitLogicCall VerifyAgainstTX failed"}
```
3. some chains for which SLCs don't reach chain since valset id is reported by pigeon as 0 instead of the actual valset id. This also means that subsequent valset updates don't go through (Gnosis)
```
{"level":"warn","module":"server","tx_hash":"0x9ebd627c2b37523c9615325638734eeb8178ba8b4b6c210e0a7ca66bcbe50f0b","valset_id":0,"updated_valset_id":1352,"time":"2024-07-15T17:05:14Z","message":"UpdateValset VerifyAgainstTX failed"}
```

Note:
Gnosis bot script (offramp-eure-bot on bots-server) was stopped, once restarted it will add more logic calls to the queue

**Comment by verabehr** [comment]
Note that #2 is not actually a blocker for the bots to be working at the moment

**Comment by verabehr** [comment]
From the latest information this most likely all comes down to an attestation issue. Unclear yet why attestation sometimes fails

**Comment by maharifu** [comment]
The issue happens when:
* Some pigeons sign the message in the queue and a pigeon picks it up
* While this pigeon is relaying the message, some more pigeons sign it
* When we try to attest it, the SignData in paloma doesn't match the SignData of the original message, since more pigeons signed it since!

**Comment by taariq** [comment]
@maharifu will you please confirm that this issue covers the failure of Gnosis relays to complete as we saw here: https://gnosisscan.io/address/0x5d5dab5c8f2b182ca96b3201d50e3b610c3250f1#internaltx

as per @verabehr 
_The last valset is not yet attested, so the next one can't go through. From the root cause analysis here, it seems that this doesn't always happen, so we got lucky the last two times and probably will again._

**Comment by verabehr** [comment]
for completeness: I was referring to the root cause analysis in the comment of this ticket https://github.com/VolumeFi/paloma/issues/1875#issuecomment-2231232481

**Comment by maharifu** [comment]
@taariq Yes, it should be the same issue. UpdateValset messages are not successfully attested, so we use the wrong valset on SLCs and they don't go through. The PR applies for both SLC and UpdateValset messages.
